resources/utilities.nix

```diff
@@ -76,4 +76,10 @@ in {
       install_name_tool -add_rpath "${libwasmvm}/lib" $out/bin/${binName}
     ''
     else null;
+  
+  darwin-deps = pkgs.lib.optional pkgs.stdenv.isDarwin (with pkgs;
+    with darwin.apple_sdk.frameworks; [
+      Security
+      SystemConfiguration
```

Comment:
Is this framework really necessary? `otool -L` does not show it for me:

```
/System/Library/Frameworks/Security.framework/Versions/A/Security (compatibility version 1.0.0, current version 60420.101.2)
/System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1971.0.0)
/usr/lib/libiconv.2.dylib (compatibility version 7.0.0, current version 7.0.0)
/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1319.100.3)
```

Reply:
Security 100% - fails to compile on m1